### PR TITLE
feat: Categorize Rust's keywords using more specific scopes

### PR DESCRIPTION
**Pull request by poliorcetics**

### CODE REVIEW

**Comment by poliorcetics** [comment]
Screenshot 2022-10-28 at 21 30 01

Allows differenciating `return` for example

File: runtime/queries/rust/highlights.scm [hunk 1]
```diff
@@ -143,24 +151,28 @@
...
 
+[
+  "struct"
+  "enum"
+  "union"
+
+  "type"
+] @keyword.storage.type
+
+"let" @keyword.local
```
Comment (sudormrfbin) [inline runtime/queries/rust/highlights.scm:174]: `keyword.local` doesn't seem to be a valid scope

Comment (poliorcetics) [reply runtime/queries/rust/highlights.scm:174]: It exists as a consequence of `keyword.local.import` in `ledger`, but yes that was not correct, moved to `keyword.storage`

**Review by sudormrfbin — APPROVED** [review]

**Comment by the-mikedavis** [comment]
Thanks!

**Pull request merged**